src/AddTodoForm.jsx

```diff
@@ -1,12 +1,20 @@
-import { useState } from 'react'
+import React from "react";
+
+function AddTodoForm(props) {
+    function handleAddTodo(event) {
+        props.onAddTodo(todoTitle);
+        event.preventDefault();
+        const todoTitle = event.target.elements.title.value;
+        console.log(todoTitle);      
```

Comment:
A couple things to note:
- The `event.preventDefault()` should come first
- You can't access variables before they're declared. Case in point 👇🏾
```js
props.onAddTodo(todoTitle)

// todoTitle doesn't exist up until down here

const todoTitle = event.target.elements.title.value
```